massifs/logdircache.go

```diff
@@ -602,13 +602,9 @@ func (d *LogDirCacheEntry) setMassifStart(opts DirCacheOptions, logfile string,
 		return fmt.Errorf("%w: header=%d, expected=%d", ErrLogFileMassifHeightHeader, ms.MassifHeight, opts.massifHeight)
 	}
 
-	// if we already have a log with the same massifIndex, read from a
-	// *different file*, we error out as the files in directories are
-	// potentially not for the same tenancy - which means the data is not
-	// correct
-	if cachedLogFile, ok := d.MassifPaths[uint64(ms.MassifIndex)]; ok && cachedLogFile != logfile {
```

Comment:
this check happens in 2 places in this file? did you manage to take it out the other place as well?

Reply:
ah thanks I will look

Reply:
huh, but the unit test for that is failing on ci :-/